lib/safira/accounts/accounts.ex

```diff
@@ -106,18 +120,31 @@ defmodule Safira.Accounts do
     |> Repo.update()
   end
 
+  def update_attendee_association(%Attendee{} = attendee, attrs) do
+    attendee
+    |> Attendee.update_changeset_discord_association(attrs)
+    |> Repo.update()
+  end
+
   def update_attendee_sign_up(%Attendee{} = attendee, attrs) do
     attendee
     |> Attendee.update_changeset_sign_up(attrs)
     |> Repo.update()
   end
 
+
```

Comment:
Not needed? Maybe run the formatted task :muscle:

Reply:
Though that was a good idea until I ran the task and half of the project's files were modified